### PR TITLE
Update Macedonian TV channels

### PR DESCRIPTION
**Pull request by DeepSpaceHarbor**
This PR will:
1. Add a new TV channel - mnet kids
2. Replace the stream URL for mrt 1, 2 and sobraniski with a direct stream URL. For some reason, the forwarder URL doesn't play the stream when I use the playlist in Kodi. The remaining streams that used the same forwarding URL are left untouched since I don't have an alternative stream for them.

### CODE REVIEW

File: lists/north_macedonia.md
Comment (KAMI911) [inline lists/north_macedonia.md:1]: Is this an EPG id?

nasatv.com.mk/cool-tv |

Comment (DeepSpaceHarbor) [reply lists/north_macedonia.md:1]: What are the rules for EPG ids? Some of the ids used the format `TvName.mk` and I replaced them with the website URL (your example being one of them). Others were without official website and left as `TvName` as EPG id.

**Comment by KAMI911** [comment]
We could use TVG-IDs found here:
https://iptv-org.github.io/

**Comment by DeepSpaceHarbor** [comment]
Thanks for the help @KAMI911 ! I've updated the EPG ids to match the ones from the link you shared.

**Comment by DeepSpaceHarbor** [comment]
Hey @TVKaista, I did a short test right now and the issue remains. I believe the problem comes from Kodi and how it handles the redirects that are coming from stream URLs. Your service does an amazing job, I'm the one who should look into alternative players and maybe to it the switch in the future.

**Comment by KAMI911** [comment]
What is the status of this issue?

**Comment by DeepSpaceHarbor** [comment]
@KAMI911  I cleaned up the branch conflicts, we should be good to go.

**Comment by KAMI911** [comment]
Thank you!

**Pull request merged**